src/Spryker/Client/RabbitMq/RabbitMqClient.php

```diff
@@ -38,4 +38,26 @@ public function getConnection(): ConnectionInterface
     {
         return $this->getFactory()->getStaticConnectionManager()->getDefaultConnection();
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @param string $queue
+     * @param string|null $storeCode
+     * @param string|null $locale
+     *
+     * @return array<string, int>
+     *@api
+     *
+     */
+    public function getQueueMetrics(
+        string $queue,
+        ?string $storeCode = null,
+        ?string $locale = null
+    ): array
+    {
+        return $this->getFactory()->createQueueMetricReader()->getQueueMetrics($queue, $storeCode, $locale);
```

Comment:
this method will be executed multiple times per second, can we "cache" QueueMetricReader into static var here or in the factory?

Reply:
added